package/libyui.changes

```diff
@@ -1,3 +1,6 @@
+- Added ui test before loading extended widget plugin, to avoid
```

Comment:
The dashy header with timestamp and email is missing. You can use `osc vc package/` to add it, or you can improvise one with 
```sh
dd if=/dev/zero status=none bs=1 count=67 | tr '\0' -; echo
echo "$(LANG=en_US date) - $USER@$(hostname -d)"
```